plat/common/include/x86/gsbase.h

```diff
@@ -0,0 +1,117 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <x86/cpu.h>
+#include <uk/essentials.h>
+
+/* TODO: Use wrgsbase/rdgsbase instead, as they are faster */
```

Comment:
I was referring to the instructions themselves. There are also dedicated instructions to read/write from/to GS/FS, because MSR read/write is slow. But we don't have those yet (you need to set a bit in CR4 AFAIK) so I thought I would simply add a TODO. Also #862